waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -59,6 +60,10 @@
   "Kubernetes reports dates in ISO8061 format, sans the milliseconds component."
   (DateTimeFormat/forPattern "yyyy-MM-dd'T'HH:mm:ss'Z'"))
 
+(def rs-version-time-format
+  "The time format used while creating ReplicaSet version labels, only alphanumeric characters."
```

Comment:
Let's make the new field an annotation rather than a label so that we can use our default iso8601 format. I don't think having it as a label will be very useful anyway since the values are so granular and unpredictable.

Reply:
Switched to using annotations.